examples/xml-from-record-conversion/xml_from_record_conversion.out

```diff
@@ -0,0 +1,2 @@
+bal run xml_from_record_conversion.bal
```

Comment:
```suggestion
$ bal run xml_from_record_conversion.bal
```

Please check and add the $ sign in all the commands added in all `.out` files.

Reply:
Do we need to add the $ sign to the master repo BBE?

Reply:
Sorry, me bad. We need not.